include/dbcppp/Signal.h

```diff
@@ -121,6 +135,12 @@ namespace dbcppp
         DBCPPP_MAKE_ITERABLE(ISignal, Receivers, std::string);
         DBCPPP_MAKE_ITERABLE(ISignal, ValueEncodingDescriptions, IValueEncodingDescription);
         DBCPPP_MAKE_ITERABLE(ISignal, AttributeValues, IAttribute);
+
+        /// \brief Mapping of this multiplexed signal to specific value ranges of a selected multiplexor switch signal.
+        /// 
+        /// In a valid DBC, this can only have 0 or 1 entries.
```

Comment:
... got it wrong. There are multiple allowed, and in that case ALL of them are required to match simultaniously.